src/components/Reports/DonationsReport/Table/DonationsReportTable.tsx

```diff
@@ -0,0 +1,252 @@
+import React, { useState } from 'react';
+import {
+  Box,
+  Typography,
+  Button,
+  Link,
+  Divider,
+  styled,
+  Table,
+  TableCell,
+  TableRow,
+} from '@material-ui/core';
+import IconButton from '@material-ui/core/IconButton';
+import EditIcon from '@material-ui/icons/Edit';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import { useTranslation } from 'react-i18next';
+import { DataGrid, GridColDef, GridCellParams } from '@material-ui/data-grid';
+import { DateTime } from 'luxon';
+import { EmptyDonationsTable } from '../../../common/EmptyDonationsTable/EmptyDonationsTable';
+
+interface Donation {
+  date: Date;
+  partnerId: string;
+  partner: string;
+  currency: string;
+  foreignCurrency: string;
+  convertedAmount: number;
+  foreignAmount: number;
+  designation: string;
+  method: string;
+  id: string;
+}
+
+interface Props {
+  data: Donation[];
+  accountListId: string;
+}
+
+const DataTable = styled(Box)(({ theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  '& .MuiDataGrid-row.Mui-even:not(:hover)': {
+    backgroundColor:
+      theme.palette.type === 'light'
+        ? theme.palette.common.white
+        : theme.palette.cruGrayLight.main,
+  },
+}));
+export const DonationsReportTable: React.FC<Props> = ({
+  data,
+  accountListId,
+}) => {
+  const { t } = useTranslation();
+  const accountCurrency = data[0].currency;
```

Comment:
This is not the cleanest way to get the currency... I hope that when we work with real data from GraphQL, there will be an easier way to access this, separate from the donations array.  But for now we can keep this.